pkg/cmd/controller/controller_build.go

```diff
@@ -967,6 +977,13 @@ func updateForStage(si *tekton.StageInfo, a *v1.PipelineActivity) {
 	}
 }
 
+// getStepTitle translates the step container's name into the title for the step used in PipelineActivity
+func getStepTitle(containerName string) string {
+	name := strings.Replace(strings.TrimPrefix(containerName, "build-step-"), "-", " ", -1)
```

Comment:
hmm, so there is no other way to get the actual step name? That seems to be quire fragile code. Maybe we step name should be label on the container?